framework/include/utils/SymmetricRankTwoTensor.h

```diff
@@ -144,7 +144,7 @@ class SymmetricRankTwoTensorTempl
 
 public:
   /// Copy assignment operator must be defined if used
-  SymmetricRankTwoTensorTempl(const SymmetricRankTwoTensorTempl<T> & a) = default;
+  // SymmetricRankTwoTensorTempl(const SymmetricRankTwoTensorTempl<T> & a) = default;
```

Comment:
Did you mean to leave this commented out, rather than removing it?

Reply:
I... can't remember. I'll just revert that line. This was during a time I was finding warnings with the new compilers. Since I've backed out of bumping the compilers, that work is no longer needed.